static/app/views/dashboardsV2/widgetBuilder/buildSteps/columnsStep/index.tsx

```diff
@@ -49,20 +49,20 @@ export function ColumnsStep({
       description={
         dataSet === DataSet.ISSUES
           ? tct(
-              '[tagFieldLink: Tag and field] columns will help you view more details about the issues (i.e. title).',
+              '[fieldTagLink: Field and tag] columns will help you view more details about the issues (e.g., title).',
               {
-                tagFieldLink: (
+                fieldTagLink: (
                   <ExternalLink href="https://docs.sentry.io/product/sentry-basics/search/searchable-properties/#event-properties" />
                 ),
               }
             )
           : tct(
-              'To group events, add [functionLink: functions] f(x) that may take in additional parameters. [tagFieldLink: Tag and field] columns will help you view more details about the events (i.e. title).',
+              'To stack events, add [functionLink: functions] f(x) that may take in additional parameters. [fieldTagLink: Field and tag] columns will help you view more details about the events (e.g., title).',
```

Comment:
nit: we are using the 'group' wording in a couple of places. shouldn't we stick to it? in the notion's doc is also group and in the sentry doc we have a phrase with `Bar charts will group results by day`

Reply:
I actually took from our documentation ~
See here: [](https://docs.sentry.io/product/discover-queries/query-builder/#stacking-functions)
When speaking of functions, we call that "stacking functions" that allow events to be "stacked"